lib/move_requester.rb

```diff
@@ -0,0 +1,29 @@
+class MoveRequester
+  def initialize(ui)
+    @ui = ui
+  end
+
+  def request(board)
+    @ui.output('Please choose a move:')
+    result = @ui.input
+    if !is_valid?(result, board)
```

Comment:
rather than negating `is_valid?` with a `!` I would suggest writing a method called `is_invalid?` - then the code is easier to read. Unless you're using `is_valid?` anywhere else already?